includes/settings/class-wcpdf-settings-debug.php

```diff
@@ -66,6 +59,9 @@ public function output( $active_section ) {
 			case 'numbers':
 				$this->display_numbers();
 				break;
+			default:
+				do_action( 'wpo_wcpdf_settings_debug_section_output', $active_section );
+				break;
 		}
 
 		do_action( 'wpo_wcpdf_settings_debug_after_output', $active_section );
```

Comment:
Maybe you could add all the sections here as well? `$sections`

Reply:
Done.